### ISSUE TITLE
Fix application source maps

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#3689]
### What should we do?

From a bunch of [sentry logs](https://camunda-modeling.sentry.io/issues/4259417224/?project=5223041&query=is%3Aunresolved+children&referrer=issue-stream&sort=user&statsPeriod=14d&stream_index=2) we can see that the sourcemaps seem to be currently broken.

Let's fix them, i.e. double-check if we still include source-maps in the right way.

### Why should we do it?

Without sourcemaps we cannot really properly root cause issues.

**Comment by nikku** [comment]
Moving to ready as this has an impact on our ability to maintain the app.

**Comment by nikku** [comment]
Let's assume this gets fixed via https://github.com/camunda/camunda-modeler/pull/3702.

**Comment by nikku** [comment]
Monitoring if this is fixed with Camunda Modeler v5.13.0

**Comment by barmac** [comment]
I believe this is still a problem in v5.13.0: https://camunda-modeling.sentry.io/issues/4182932803/events/ca6a248245cc46d6a5c9a5fc1509d004/?project=5223041

**Comment by marstamm** [comment]
**investigation**

TL;DR: Source Maps work, integration with sentry is broken,

Long version:

I could not find a version on Sentry that had working sourcemaps, so I assume it never worked before. (e.g. 5.0.0 release: https://camunda-modeling.sentry.io/issues/4199220065/?project=5223041&query=release%3A5.0.0&referrer=release-issue-stream)

I validated that the source-maps work fine locally on the artifacts. 

Source maps not being available on sentry seems to be a configuration problem, which is also indicated by the error shown on the report
![image](https://github.com/camunda/camunda-modeler/assets/21984219/66474786-19cd-49b1-9696-1e2ee6d47d3f)

I'll check if I can see what is uploaded/checked against during the build process

**Comment by marstamm** [comment]
Since the 5.13 release, no source-map artifacts are being uploaded:

![image](https://github.com/camunda/camunda-modeler/assets/21984219/c744a73e-0462-412a-9f6e-fb991d1ff2a6)

However, even in 5.12.1, the uploaded artifacts are not used in the stack traces: https://camunda-modeling.sentry.io/issues/4252567575/?project=5223041&query=release%3A5.12.1&referrer=release-issue-stream
I expect this to be a problem because the path of the reported error starts with `~/build` instead of just `/build`

https://github.com/getsentry/sentry-webpack-plugin/issues/228

**Comment by nikku** [comment]
@marstamm We could use the nightly or an on demand build to check sentry, indeed.

Alternatively we could double-check if the current build works locally with source-maps. If that is already broken, then we know it won't work for Sentry (after upload) either.

**Comment by marstamm** [comment]
Sourcemaps work fine and were never broken. They just don't get uploaded to Sentry anymore and are also referenced wrong. Both changes need to be done on our sentry configuration and reporting (cf. https://github.com/camunda/camunda-modeler/tree/3689-fix-sentry-source-map)

**Comment by nikku** [comment]
Sounds great :slightly_smiling_face:

**Comment by marstamm** [comment]
It seems like build on demand does not create a sentry release (cf. https://camunda-modeling.sentry.io/releases/?project=5223041). Not sure if this is expected or if the creation of the project is broken as well. Need to investigate this further

**Comment by nikku** [comment]
I propose we test this on the nightly. It is a simple enough change to iterate on.

**Comment by marstamm** [comment]
![image](https://github.com/camunda/camunda-modeler/assets/21984219/9fac585a-75c7-44ec-82ca-9154b97763ea)

No release created for the nightly, meaning the Source map upload is still broken. I'll try to create a release locally to debug this without having to wait for nightlies
